python/pyspark/sql/__init__.py

```diff
@@ -41,6 +41,13 @@
 """
 from __future__ import absolute_import
 
+
+def since(version):
```

Comment:
I'v planned to move this into pyspark/__init__.py later, let's focus on SQL API in this PR.